frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -10,9 +10,9 @@ export interface AccountBasicInfo {
   deletedAtBlockTimestamp: number | null;
 }
 
-interface AccountStats {
-  inTransactionsCount: number;
-  outTransactionsCount: number;
+export interface AccountStats {
+  inTransactionsCount?: number;
+  outTransactionsCount?: number;
```

Comment:
1. I feel `?` is not necessary since you either have the TransactionsStats or not have them at all
2. `Account` type (defined below) should not have `AccountStats`